frontend/src/pages/SkillsMap/StackMap/CourseMapForCurrentStack/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { EmployeeRank } from 'enums/employee';
+
+export const getRankNameByRankId = (ranklId: EmployeeRank): string => {
+  const rankNames = {
+    [EmployeeRank.Junior]: 'Junior Rank',
+    [EmployeeRank.Middle]: 'Middle Rank',
+    [EmployeeRank.Senior]: 'Senior Rank',
+  };
```

Comment:
do we really need to create separate utils file for this function? or maybe it can be moved to container?